test/parallel/test-sqlite.js

```diff
@@ -80,6 +83,84 @@ suite('DatabaseSync() constructor', () => {
   });
 });
 
+suite('DatabaseSync.prototype.loadExtension()', () => {
```

Comment:
Can you add the tests to a new test file to keep this one from getting too large (it can lead to timeouts on slower CI machines).